shimmingtoolbox/masking/shape.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Image mask with shape API
+"""
+
+import numpy as np
+
+
+def shape(data, shape, center_x=None, center_y=None, center_z=None, len_x=None, len_y=None, len_z=None, radius=None):
+    """
+    Wrapper to different shape masking algos
+
+    Args:
+
+    Returns:
+        nd.array: mask
+    """
+
+    # allowed_shapes = {'square', 'cube', 'disk', 'ball'}
+
+    # Make sure input params are valid
+    if data.ndim not in [2, 3]:
```

Comment:
If you check dimensions in your masking function afterward, this line becomes redundant and prone to mistake afterward.